libopenage/console/buf.cpp

```diff
@@ -617,6 +617,7 @@ void Buf::process_escaped_cp(int cp) {
 	switch (cp) {
 	case 'c': //RIS (full reset)
 		this->reset();
+		break;
```

Comment:
This changes the logic, @mic-e pls help how the cp needs to be processed. I guess the new `break` is correct, though.